cmd/contour/serve.go

```diff
@@ -193,9 +199,26 @@ func NewServer(log logrus.FieldLogger, ctx *serveContext) (*Server, error) {
 	}
 
 	// Instantiate a controller-runtime manager.
-	mgr, err := manager.New(restConfig, manager.Options{
+	options := manager.Options{
 		Scheme: scheme,
-	})
+	}
+	if !ctx.DisableLeaderElection {
```

Comment:
nit: Just a thought, maybe handle the positive case and log that leader election is disabled? Then the else can set the leader election config to reduce the stutter with not-disable-leader-election?

```suggestion
	if ctx.DisableLeaderElection {
	 <log disabled>
	 options.LeaderElection = false
	} else {
	 options.LeaderElection = true
	 ...	
	}
```

Reply:
ah yeah logging leader election is disabled would be good, the leader election code in controller runtime does some logging so we could rely on that or also add a log for the enabled case too if we want